spec/fixtures/MondrianTest.xml

```diff
@@ -41,7 +41,7 @@
     </Hierarchy>
   </Dimension>
 
-<Cube name="Sales" defaultMeasure="Unit Sales">
+<Cube name="Sales $sales" defaultMeasure="Unit Sales"  visible="false">
```

Comment:
Do not do such radical change by changing the Sales cube from a normal cube to a virtual cube.